test/parallel/test-fs-watch-close-when-destroyed.js

```diff
@@ -5,6 +5,13 @@
 
 const common = require('../common');
 
+// fs-watch on folders have limited capability in AIX.
+// The testcase makes use of folder watching, and causes
+// hang. This behavior is documented. Skip this for AIX.
+
+if (common.isAIX)
```

Comment:
how is it that this test passed prior to this PR? are we breaking an existing feature?

Reply:
Before this, we were just rejecting all OS except Windows and macOS on recursive fs watch tests.

Referring the code on test-fs-watch-recursive.js: 
```if (!common.isOSX && !common.isWindows) {```

Reply:
but this file specifically runs succesfully on aix prior to this PR?

Reply:
This file does not trigger any changes on the watched file/folder path. That's why it runs successfully on AIX, and creates the illusion that it works on AIX. 

**My assumption is:** There is no need to test `close()` function if folder watch is not supported on AIX.